2. Xizmatlar/2.1. Asoslar/Asoslar.md

```diff
@@ -0,0 +1,65 @@
+# 2.1 Asos Xizmatlari (Brokerga O'zaro Bog'langan)
+
+## 2.1.0 Kirish
+
+Asos xizmatlari bu sizning ishingizning mantig'i(biznes logika) va brokerlar o'rtasidagi birinchi bog'lanish nuqtasi hisoblanadi.
+
+Umimiy qilib aytganda, yuqori darajadagi ish mantig'i(biznes logika) qayerda sodir bo'lsa, usha holatda ish mantig'i(biznes logika) va mavhumlik qatlami jarayonlarni qayta ishlash uchun birlashishi, brokerga o'zaro bog'liq xizmatlari bo'ladi. Keyingi bo'limda qayta ishlovchi xizmatlarni yaratishni boshlaganimizda, yuqori darajadagi ish mantig'i(biznes logika) haqaida gaplashamiz.
+
+Brokerga o'zaro bog'liq xizmatlarining asosiy vazifasi, tizim orqali kiruvchi va chiquvchi ma'lumotlarning tizimli, mantiqiy va tashqi tomondan tasdiqlanganligini va tekshirilganligini ta'minlashdir.
+
+Shuningdek, brokerga o'zaro bog'liq xizmatlar sodda jarayonlar ustida tekshirish qatlami sifatida brokerlar allaqachon taklif qilgananini o'ylashingiz mumkin.
```

Comment:
`You can also think of broker-neighboring services as a layer of validation on top of the primitive operations the brokers already offer.` bu gapni tarjima qilganingizda asosiy g'oyani anglab ola olmadim. Quyidagi o'zgarishni taklif qilaman:

`Shuningdek, brokerga o'zaro bog'liq xizmatlarni broker taqdim etadigan sodda amallar ustida boshlang'ich tekshiruvni amalga oshiruvchi xizmatlar deb qarashingiz ham mumkin.`